pyspecdata/core.py

```diff
@@ -1999,6 +2002,9 @@ def div_units(self, arg1, arg2=None):
         or `d.div_units("s")` to divide the
         data units by seconds.
 
+        In other words, if you pass "a" and the units of your data are
+        in "b", then this returns x, such that (x a)/(b) = 1.
+
```

Comment:
we did this together when reviewing Verify.py and this helps clarify the use of div_units (could help us understand why we multiply vs divide by this return

Reply:
yes